doc/replication.md

```diff
@@ -71,11 +71,11 @@ From the device side, replication is straightforward. You can replicate from a
 local datastore to a remote database, from a remote database to a local
 datastore, or both ways to implement synchronisation.
 
-Replications are set up in code on a device. Use `PullReplication` and
-`PushReplication` objects to create pre-configured `Replicator` objects
-using a `ReplicatorFactory`. Then call `start()` on the `Replicator` object
-to start a replication. Each `Replicator` object can be assigned a `Listener`
-to receive messages when replication completes or encounters an error.
+Replications are set up in code on a device. Use `ReplicatorBuilder` to create 
+a pre-configured `Replicator` object. Then call `start()` on the `Replicator`. However
```

Comment:
"As a convenience, it is possible to call start() directly on the `ReplicatorBuilder` which builds and starts a replication"